scripts/build/TestPlatform.Dependencies.props

```diff
@@ -6,8 +6,8 @@
     <!-- Name of the elements must be in sync with test\Microsoft.TestPlatform.TestUtilities\IntegrationTestBase.cs -->
     <NETTestSdkPreviousVersion>15.5.0</NETTestSdkPreviousVersion>
 
-    <MSTestFrameworkVersion>1.3.2</MSTestFrameworkVersion>
-    <MSTestAdapterVersion>1.3.2</MSTestAdapterVersion>
+    <MSTestFrameworkVersion>1.3.1</MSTestFrameworkVersion>
```

Comment:
This change required to show proper diagnostic info on test failures of vstest repo. Related to https://github.com/Microsoft/testfx/issues/451 . We should fix the trx parsing logic in vstest repo.

![datadriven_h](https://user-images.githubusercontent.com/4337332/41832016-b55d387c-7867-11e8-9e6b-43da4cdc7e34.PNG)